.github/workflows/test-frontend-unit.yml

```diff
@@ -0,0 +1,54 @@
+name: "Frontend test suite"
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - dev
+      - release/*
+    paths:
+      - '**/frontend/**/*.ts'
+      - '**/frontend/**/*.js'
+      - '**/frontend/**/*.json'
+
+  pull_request:
+    types: [opened, reopened, synchronize]
+    paths:
+      - '**/frontend/**/*.ts'
+      - '**/frontend/**/*.js'
+      - '**/frontend/**/*.json'
+
+permissions:
+  contents: read
+
+defaults:
+  run:
+    working-directory: ./frontend
+
+jobs:
+  units:
+    name: Units
+    runs-on: ubuntu-latest
```

Comment:
`ubuntu-latest` resolves to GitHub's Ubuntu 24.04 runner image, which [includes Chrome, chromedriver amongst other browsers and drivers](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#browsers-and-drivers).

Reply:
It might be better to explicitly specify `ubuntu-24.04`:
![image](https://github.com/user-attachments/assets/e5f97d54-3eac-480d-9186-bcd5195c0320)

Reply:
I don't think we depend on the specific version here, so let's keep this as is